src/scripts/i18n/list_source_files.py

```
@@ -0,0 +1,113 @@
+    parser.add_argument(
+        "--types",
+        required=True,
+        help=f"Comma-separated list of file categories to include. Available: {ALL_TYPES}",
+    )
+    args = parser.parse_args()
+
+    types_requested = [t.strip() for t in args.types.split(",") if t.strip()]
+
+    unknown = [t for t in types_requested if t not in SUFFIX_MAP]
+    if unknown:
+        raise SystemExit(f"Unknown type(s): {', '.join(unknown)}. Valid values: {ALL_TYPES}")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)